### PR TITLE
PAS-378 | Create pull request template

### PR DESCRIPTION
**Pull request by merissaacosta**
### Ticket

- Closes [PAS-378](https://bitwarden.atlassian.net/browse/PAS-378)
### Description

This is a new PR template that will help the team communicate changes to each other which speeds up the review process. 
### Shape

My approach with this template is to make it easier for PR reviewers to understand better what they are reviewing and to promote "Shape" PR's to share architectural decisions. 
### Screenshots

(screenshots would go here)
### Checklist
I did the following to ensure that my changes were tested thoroughly:
- [ ] example 1
- [ ] example 2

I did the following to ensure that my changes did not introduce new security vulnerabilities:
- [ ] example 1
- [ ] example 2

[PAS-378]: https://bitwarden.atlassian.net/browse/PAS-378?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<!--
+Thank you for contributing! Please follow the steps below to help us process your PR quickly.
+- 📝 Use a meaningful title for the pull request.
+- 💭 Write a clear description and share screenshots (if applicable) to help describe your change.
+- 🔍 Not all sections below will apply to you and are mostly for our internal team. It's okay to delete them if they are not applicable. 
+-->
```
Comment (merissaacosta) [inline pull_request_template.md:6]: @Tyrrrz what do you think of this?

File: pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+Give a brief overview of your approach, the pros/cons that were considered, etc.
+
+ -->
+
+### Screenshots
+<!-- Include any screenshots if this is a UI change. Delete if N/A. -->
+
+I did the following to ensure that my changes were tested thoroughly to ensure safety:
+[] __
+[] __
```
Comment (Tyrrrz) [inline pull_request_template.md]:
```suggestion
### Ticket

<!--
    Link the corresponding GitHub Issue(s) and/or Jira Task(s)
    that are addressed by this pull request.
-->

<!-- For GitHub Issues: -->
- Closes #xxx

<!-- For Jira Tasks: -->
<!-- (remove if external contributor) -->
- Closes [PAS-XXX](https://bitwarden.atlassian.net/browse/PAS-XXX)

### Description

<!--
    Introduction that should allow the reviewer to quickly be able to understand the reason for opening this PR.
-->

### Shape

<!--
    Give the high-level overview of the technical design involved in the implemented changes.
    If the changes don't have any architectural impact, you can remove this section.
-->

### Screenshots

<!--
    Include any relevant UI screenshots showcasing the before & after of your changes.
    If the changes don't have an UI impact, you can remove this section.
-->

### Checklist

I did the following to ensure that my changes were tested thoroughly:
- [ ] __
- [ ] __

I did the following to ensure that my changes did not introduce new security vulnerabilities:
- [ ] __
- [ ] __
```

File: pull_request_template.md (hunk 1 above)
Comment (Tyrrrz) [reply pull_request_template.md:6]: This is a nice addition 🙂

File: pull_request_template.md (hunk 2 above)
Comment (Tyrrrz) [reply pull_request_template.md]: I don't know if multiline HTML comments work, but thought it looked nicer

Comment (merissaacosta) [reply pull_request_template.md]: they work - thank you

Comment (merissaacosta) [reply pull_request_template.md]: @Tyrrrz i updated the template with these changes!

**Review by Tyrrrz — APPROVED** [review]

**Review by jonashendrickx — APPROVED** [review]

**Pull request merged**